code/modules/goonchat/browserassets/js/browserOutput.js

```diff
@@ -606,7 +636,15 @@ $(function() {
 		opts.updatedVolume = newVolume;
 		sendVolumeUpdate();
 		internalOutput('<span class="internal boldnshit">Loaded music volume of: '+savedConfig.smusicVolume+'</span>', 'internal');
-	} else {
+	} 
+	if (savedConfig.smessagecombining) {
+		if (savedConfig.smessagecombining == 'false') {
+			opts.messageCombining = false;
+		} else {
+			opts.messageCombining = true;
+		}
+	}
+	else {
 		$('#adminMusic').prop('volume', opts.defaultMusicVolume / 100);
```

Comment:
You added a bug here by tying admin music volume to `savedConfig.smessagecombining`.

Also, `opts.messageCombining = !(savedConfig.smessagecombining === 'false')` would save you a few lines of unneeded code above.

Reply:
Good catch, will fix this in a moment.